docs/mining/proof-of-work.md

```diff
@@ -7,21 +7,31 @@
 Proof-of-work mining, more commonly referred to as PoW mining, is the activity
 of committing your computer’s hardware and resources to process network
 transactions and build the blocks that make up the blockchain in the Decred
-network. Each time a block is created (by a miner), about 30 new Decred coins
-are made. These coins are then split up as follows:
+network. Each time a block is created (by a miner), which is then approved by 
```

Comment:
The 30% of the block reward which is allocated to the PoS voters is actually created regardless of whether they vote to approve or reject the block. If the 30% was not created, the voters would be incentivized to never reject a block.

The first answer under this video from @davecgh explains it well, it might actually be worth just copy and pasting part/all of this.
 
 https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/

Reply:
Theres also a second reply from Dave under his first one with some more details.

Reply:
I decided to not include further detail from @davecgh's post. I think that PoW miners may not want that level of detail on the PoW page - and it is included on other advanced pages. If you think that extra detail would be good to add here though, can work it in.